modules/m_away.c

```diff
@@ -127,6 +132,14 @@ m_away(struct MsgBuf *msgbuf_p, struct Client *client_p, struct Client *source_p
 						    source_p->username,
 						    source_p->host,
 						    source_p->user->away);
+		struct monitor *monptr = find_monitor(source_p->name, 0);
+		if(monptr)
+			sendto_monitor_with_capability_butserial(source_p, monptr, CLICAP_EXTENDED_MONITOR | CLICAP_AWAY_NOTIFY, NOCAPS, true,
```

Comment:
Adjust the regular away-notify to exclude extended-monitor (due to a GitHub limitation, I can't comment on that line).  Eliminate current_serial abuse

```suggestion
			sendto_monitor_with_capability(source_p, monptr, CLICAP_EXTENDED_MONITOR | CLICAP_AWAY_NOTIFY, NOCAPS,
```